framework/i18n/Formatter.php

```diff
@@ -787,6 +787,50 @@ public function asRelativeTime($value, $referenceTime = null)
         }
     }
 
+    /**
+     * Formats the value as a time interval.
+     * @param mixed $value the value to be formatted, in ISO8601 format.
```

Comment:
For example we have a stored phone call. We have 2 use cases, how we can show it's duration.

* We store a real duration of call in seconds.
* We have start and end datetime and every time calculate the difference between 2 dates.

For 2'nd case, you have to pass 2 dates to calculate the duration OR you have to prepare the duration in seconds and use it.

I think, that seconds is more correct way to operate with intervals